Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_ITSM_Enrichment.yml

```diff
@@ -0,0 +1,471 @@
+      description: Set the beginning of a key used for querying ServiceNow using SN descriptions and a timestamp.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '26'
+    scriptarguments:
+      key:
+        simple: sn_query
+      value:
+        complex:
+          root: inputs.search_terms
+          transformers:
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: ^ORshort_descriptionLIKE
+              suffix: {}
+          - operator: join
+            args:
+              separator: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: TimeNow
+                iscontext: true
+              suffix: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: ^createdON>=
+              suffix: {}
+    separatecontext: false
+    continueonerrortype: ''
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": -160
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '26':
+    id: '26'
+    taskid: 8aa4e3af-7655-44b1-84b7-5f7e888f8345
+    type: regular
+    task:
+      id: 8aa4e3af-7655-44b1-84b7-5f7e888f8345
+      version: -1
+      name: Build ServiceNow query - short description
+      description: Set the remaining query value with SN short descriptions for querying ServiceNow.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '5'
+    scriptarguments:
+      key:
+        simple: sn_query
+      value:
+        complex:
+          root: inputs.search_terms
+          transformers:
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: ^ORdescriptionLIKE
+              suffix: {}
+          - operator: join
+            args:
+              separator: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: sn_query
+                iscontext: true
+              suffix: {}
+          - operator: replace
+            args:
+              limit:
+                value:
+                  simple: '1'
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: OR
+    separatecontext: false
+    continueonerrortype: ''
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '27':
```

Comment:
@BigEasyJ , in this task (or a separate task) we need to make sure SNOW-ITSM integration is enabled or there is no reason to continue.